LaLauncher/src/main/java/com/github/postapczuk/lalauncher/AppsActivity.java

```diff
@@ -90,6 +101,14 @@ public void onClickHandler() {
     @TargetApi(Build.VERSION_CODES.GINGERBREAD)
     public void onLongPressHandler() {
         listView.setOnItemLongClickListener((parent, view, position, id) -> {
+            TextView selectedItem = view.findViewById(view.getId());
```

Comment:
duplicated code - you could extract it to method in AppsActivity, and use it in both places